app/views/my/tracks/_core_exercises.html.haml

```diff
@@ -21,13 +21,11 @@
                 .topic= topic_name
           .blurb= exercise.blurb
 
-        -if @side_exercises_and_solutions_by_unlocked_by[exercise.id].present? || (exercise.auto_approve? && @side_exercises_and_solutions_by_unlocked_by[nil].present?)
+        - if exercise.unlocked_exercises.any?
           .unlocked-exercises-section
             %h3 You've unlocked extra exercises!
             .unlocked-exercises
-              -if exercise.auto_approve?
-                =render "unlocked_exercises", unlocked_exercises_and_solutions: @side_exercises_and_solutions_by_unlocked_by[nil][0,10]
```

Comment:
Have we maintained this line somewhere? (ie if it's auto-approved we *also* show the first 10 side exercises with `unlocked_by: nil`)